templates/analytics/activity.html

```diff
@@ -17,15 +17,13 @@
 
```

Comment:
We can cut the `is_home` check here, too, right?

Reply:
(I think we could also cut some of the unnecessary blank lines in this file).

Reply:
Updated for both notes above.